AAEmu.Game/Models/Game/Char/Character.cs

```diff
@@ -1670,6 +1725,8 @@ public static Character Load(MySqlConnection connection, uint characterId)
                             character.Hp = character.MaxHp;
                         if (character.Mp > character.MaxMp)
                             character.Mp = character.MaxMp;
+                        if (character.Breath < character.MaxBreath)
+                            character.Breath = character.MaxBreath;
```

Comment:
Breath should be = to MaxBreath by default to avoid having to do it this way, we don't need to save breath to DB -> shouldn't be in Load